content/kubernetes-backing-services/ingress/examples/nginx-ingess-helmfile.yaml

```diff
@@ -0,0 +1,33 @@
+# https://github.com/cloudposse/helmfiles/blob/master/library/nginx-ingress.yaml
```

Comment:
I think this is taking it too far. We cannot reasonably maintain 2 copies of helmfile definitions. Do we absolutely need to include this? Can we reduce scope of the documentation to avoid needing to do this?

Reply:
I think we should be linking to geodesic git repo instead.

Reply:
Done